### DISCUSSION TITLE
EID8016

### DISCUSSION THREAD

**Discussion by OR13** [OR13/errata-discuss#5] [Errata]
# EID8016

### [AD Verify Link](https://www.rfc-editor.org/verify_errata_select.php?eid=8016)

~~~ diff
-     <...> In fact, power has become such a prominent issue in
-    modern Long Term Evolution (LTE) devices that mobile browsers close
-    HTTP connections within seconds or even immediately [SOUDERS11].
-
+     <...> In fact, power has become such a prominent issue in
+    3G mobile devices that mobile browsers close HTTP connections
+    within seconds or even immediately [SOUDERS11].
+
~~~

### Notes

Reading the reference: It mentions 3G exclusively.
3G data networks are circuit-switched so energy consumption on idle connections is an issue.
4G/5G (LTE) networks are packet-switched, and radio energy consumption (active carrier?) might not be an issue. 4G and 5G are not mentioned in the reference.

**Comment by ajeanmahoney** [comment]
[SOUDERS11] does mention 4G networks, and at the time of its writing (2011), LTE had been deployed by some carriers, but agree that the info in [SOUDERS11] to which the RFC refers is about 3G devices.

**Reply by OR13** [reply]
Based on your comment here, I might propose this errata be rejected, do you agree?

**Reply by ajeanmahoney** [reply]
[SOUDERS11] talks about 3G devices, but also has data from networks that are running 3G or 3G/4G networks. Maybe it should say "modern 3G and Long Term Evolution (LTE) devices", but it's hard to tell because the references within [SOUDERS11] are gone or paywalled. Reject because a better fix would be to provide a better reference than [SOUDERS11], but we don't know what that is?
